src/bindings/jna/libelektra4j/src/main/java/org/libelektra/KeySet.java

```diff
@@ -43,7 +43,7 @@ protected KeySet(final long p) {
 	 *
 	 * @param p Pointer to another KeySet
 	 */
-	protected KeySet(final Pointer p) {
+	public KeySet(final Pointer p) {
```

Comment:
Because I moved the NativeElektraPlugin into the plugin package and it makes usage of this method. It does not harm the development experience though if this is public.

Reply:
I did not check this, I can throw a RunTime Exception if the pointer is null but apart from that I do not see any other way around making the constructor public so that the NativePlugin can load

Reply:
Not ideal but if your design needs it you can add the check. Please document invariants.